examples/units_and_coordinates/STEREOSECCHI_starfield.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""
+===========================================================
+Identifying stars in a STEREO/SECCHI COR2 coronagraph image
+===========================================================
+
+How to use the `Vizier star catalog <https://vizier.u-strasbg.fr>`_ to
+identify bright points in a STEREO/SECCHI COR 2 image as stars.
+As a bonus, we also identify Mars.
+"""
+import matplotlib.pyplot as plt
+
+# astroquery is not a dependency of SunPy so will likely need to be
+# installed.
+from astroquery.vizier import Vizier
+import astropy.units as u
+from astropy.coordinates import SkyCoord, get_body_barycentric
+
+import sunpy.map
+from sunpy.coordinates import get_body_heliographic_stonyhurst, frames
+from sunpy.net import helioviewer
+
+###############################################################################
+# Let's download a STEREO-A SECCHI COR2 image from Helioviewer which provide
+# pre-processed images and load it into a Map.
+hv = helioviewer.HelioviewerClient()
+f = hv.download_jp2('2014/05/15 07:54', observatory='STEREO_A', instrument='SECCHI', detector='COR2')
+cor2 = sunpy.map.Map(f)
+
+###############################################################################
+# To efficiently search the star field we need to know what stars are near the
+# Sun as observed by STEREO. We need the vector that points from STEREO to the Sun.
+# By converting to HCRS we get the vector from the Sun to STEREO
+sun_to_stereo = cor2.observer_coordinate.transform_to('hcrs')
+
+###############################################################################
+# We next reflect the vector to get our search vector which points from STEREO
+# to the Sun
+stereo_to_sun = SkyCoord(ra=sun_to_stereo.ra + 180.*u.deg, dec=-sun_to_stereo.dec, distance=sun_to_stereo.distance,
```

Comment:
```suggestion
stereo_to_sun = SkyCoord(-sun_to_stereo.data, obstime=sun_to_stereo.obstime,
```
The easiest way to do this is actually `stereo_to_sun = sun_to_stereo._apply(np.negative)` because that avoids explicitly creating a new `SkyCoord`, but that requires calling a private function

Reply:
Fixed.